__tests__/__snapshots__/plugin.test.ts.snap

```diff
@@ -58,14 +66,14 @@ var f = function () {
 }();"
 `;
 
-exports[`babel plugin doesn't remove nested 'worklets' 1`] = `
+exports[`babel plugin doesn't nest worklets for UI thread 1`] = `
```

Comment:
Let's remove "UI thread" because soon you will be able to run worklets on different threads as well 😄

Reply:
😵